apps/ledger-live-desktop/src/renderer/screens/exchange/Swap2/utils/index.ts

```diff
@@ -3,19 +3,16 @@ import { useHistory } from "react-router-dom";
 import { SwapExchangeRateAmountTooLow } from "@ledgerhq/live-common/errors";
 import { NotEnoughBalance } from "@ledgerhq/errors";
 import { track } from "~/renderer/analytics/segment";
-import useFeature from "@ledgerhq/live-common/featureFlags/useFeature";
 
 export const SWAP_VERSION = "2.35";
 
 export const useGetSwapTrackingProperties = () => {
-  const ptxSwapMoonpayProviderFlag = useFeature("ptxSwapMoonpayProvider");
   return useMemo(
     () => ({
       swapVersion: SWAP_VERSION,
       flow: "swap",
-      ptxSwapMoonpayProviderEnabled: !!ptxSwapMoonpayProviderFlag?.enabled,
     }),
-    [ptxSwapMoonpayProviderFlag?.enabled],
+    [],
```

Comment:
As now it's a const I think that you can extract the object outside the hook and return the newly created const object from the hook, so you don't need to memo, wdyt?

Reply:
Yup, agreed 👍